src/main/kotlin/no/nav/syfo/behandler/api/person/BehandlerDialogmeldingPersonDTO.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.syfo.behandler.api.person
+
+data class BehandlerDialogmeldingPersonDTO(
```

Comment:
Jeg sliter med å skjønne hva denne klassen er basert på navnet. Det er ingenting i den som ser ut til å ha noe med dialogmelding å gjøre? Jeg tror mest dette er en behandler, fordi det har kontor i seg, men jeg skjønner ikke hvor Person kommer inn. Hvis dette er en behandler, kan den hete det? BehandlerDTO?

Reply:
Den kan vel i grunn hete `BehandlerDTO`. grunnen til at jeg la på et postfix og laget egen klasse var fordi jeg tenkte det er kjekt med egen DTO-klasse i Person-API'et og ikke gjenbruke eksisterende DTO(`BehandlerDialogmeldingDTO`).
Må jo ikke være slik, men den kan være grunner til at vi ikke vil sende med noen typer data til person vs til intern bruker(e.g. veileder).

Når det gjelder navnet `BehandlerDialogmeldingPersonDTO` så kan dette evt. endres til `PersonBehandlerDTO`(ref den andre kommentaren din om rekkefølge på ordene).

Reply:
Endret denne til `PersonBehandlerDTO`.

Reply:
@andersrognstad jeg endret for koden i dette API'et, så kan vi evt. endre navngivning i Veileder-API'et senere ja.